### ISSUE TITLE
[Bug Report] el-avatar become too big because undefined --el-avatar-size

### ISSUE THREAD

**Issue by jasonren0403** [element-plus/element-plus#5677]
### Element Plus version
1.3.0-beta.9

### OS/Browsers version
Edge latest

### Vue version
3.2.29

### Reproduction Link
https://codepen.io/jasonren0403/pen/bGYdbox

### Steps to reproduce
Insert an `<el-avatar>` in `<el-card>`'s header slot(Or maybe just insert an `el-avatar` into the document)
Playground repro code below:
```
<script setup lang="ts">
import { ref } from 'vue'
import { setupElementPlus } from './element-plus.js'

// setup for element plus, don't remove.
setupElementPlus()

</script>

<template>
	<el-card>
    <template #header>
      <el-avatar src="https://cube.elemecdn.com/0/88/03b0d39583f48206768a7534e55bcpng.png" size="small"></el-avatar>
    </template>
    <div>
      blahblah
    </div>
  </el-card>
  <div>
    body after card
  </div>
</template>
```
A more simple repro code:
```
<script setup lang="ts">
import { ref } from 'vue'
import { setupElementPlus } from './element-plus.js'
import { User } from '@element-plus/icons-vue'

// setup for element plus, don't remove.
setupElementPlus()

const msg = ref('Hello World!')
</script>

<template>
      <el-avatar src="https://cube.elemecdn.com/0/88/03b0d39583f48206768a7534e55bcpng.png" size="small"></el-avatar>
</template>
```

### What is Expected?
--el-avatar-size should not be undefined

### What is actually happening?
--el-avatar-size is undefined
![image](https://user-images.githubusercontent.com/40999116/151399798-050f8502-d27f-4dd1-90ef-0337a14d1991.png)

**Comment by cn-troy** [comment]
has PR #5680

**Comment by kooriookami** [comment]
Fixed in #5583
